<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_page</class>
 <widget class="QMainWindow" name="main_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>591</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>151</width>
       <height>511</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/backgournds/FlyTranslate-logo/background_image.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-20</y>
       <width>151</width>
       <height>111</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/backgournds/FlyTranslate-logo/logo_only_2.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMainButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Main Page</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRunTimeSettings">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Runtime Setting</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnProfileSettings">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Profile Setting</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFAQ">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>240</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>FAQ</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLogout">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>535</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>  Log out</string>
     </property>
     <property name="icon">
      <iconset resource="../Images_for_design/static_img.qrc">
       <normaloff>:/logout/logout.png</normaloff>:/logout/logout.png</iconset>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFeedback">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>290</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Feedback</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>0</y>
      <width>641</width>
      <height>591</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell</family>
       <pointsize>11</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Welcome Back,</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblUsername">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>431</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SAMPLEUSERNAME!!</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>811</width>
       <height>591</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/backgournds/FlyTranslate-logo/background_image.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="cbbSource">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>270</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbbTarget">
     <property name="geometry">
      <rect>
       <x>385</x>
       <y>270</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>385</x>
       <y>240</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Target Language:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblbtnSwapLanguage">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>270</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../Images_for_design/interactable_image_source.qrc">:/can_switch/swap.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>240</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Source Language:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSaveConfig">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>340</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Save current configurations</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCropImage">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Crop Area</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>621</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <strikeout>true</strikeout>
      </font>
     </property>
     <property name="text">
      <string>----------------------------------------------------------------------------------------------------------------------------</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMinimize">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>510</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Minimize application</string>
     </property>
     <property name="icon">
      <iconset resource="../Images_for_design/interactable_image_source.qrc">
       <normaloff>:/minimize/minimize.png</normaloff>:/minimize/minimize.png</iconset>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblTipsLang">
     <property name="geometry">
      <rect>
       <x>575</x>
       <y>270</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/tips/information.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>420</y>
       <width>581</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;To translate screen text, minimize application first for best experience: &lt;/p&gt;&lt;p&gt;(It will hang for a while if it's initializing or downloading language resources) (No need to force stop if it hangs, please wait for the program to initialize.)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Scan full screen without crop:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbToggleScanFullScreen">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>100</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox::indicator:unchecked{
	image: url(:/not-toggled/off-button.png);
}

QCheckBox::indicator:checked{
	image: url(:/toggled/on-button.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnTestOutputImageSize">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>160</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Preview output 
image size</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblTipsScan">
     <property name="geometry">
      <rect>
       <x>575</x>
       <y>170</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/tips/information.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_2</zorder>
    <zorder>label_4</zorder>
    <zorder>lblUsername</zorder>
    <zorder>cbbSource</zorder>
    <zorder>cbbTarget</zorder>
    <zorder>label_5</zorder>
    <zorder>lblbtnSwapLanguage</zorder>
    <zorder>label_3</zorder>
    <zorder>btnSaveConfig</zorder>
    <zorder>btnCropImage</zorder>
    <zorder>label_11</zorder>
    <zorder>btnMinimize</zorder>
    <zorder>lblTipsLang</zorder>
    <zorder>label_8</zorder>
    <zorder>label_9</zorder>
    <zorder>cbToggleScanFullScreen</zorder>
    <zorder>btnTestOutputImageSize</zorder>
    <zorder>lblTipsScan</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Images_for_design/background_imgs.qrc"/>
  <include location="../Images_for_design/static_img.qrc"/>
  <include location="../Images_for_design/interactable_image_source.qrc"/>
 </resources>
 <connections/>
</ui>
